README.md

```diff
@@ -1,55 +1,37 @@
 # iRODS TypeScript HTTP API Wrapper
 
-## Installation and Usage
-
-1. Install npm package: 
+## Installation
 
 ```
 npm install git+https://github.com/irods/irods_client_http_typescript
 ```
 
-2. Import `IrodsHttpClient` class:
+## Usage
+
+After creating an instance of the wrapper, you can authenticate it within an asynchronous context, then do any operations available from the http api. The following code demonstrates a way of creating a new collection with the wrapper.
 
 ```js
 import { IrodsHttpClient } from 'irods_client_http_typescript'
-```
+import type { URLComponentsType } from 'irods_client_http_typescript'
 
-3. Create a url configuration object, containing info about the irods http api that's running on your local machine:
-
-```js
+// Create a url configuration object, containing info about
+// the irods http api that's running on your local machine.
 const urlComponents: URLComponentsType = {
     host: '<host>',
     port: '<port>',
     version: '<version>',
 }
```

Comment:
If the person managing the HTTP API decides to hide the port and version behind a proxy, then the idea of separating host, port, and version falls apart.

`URLComponentsType` can be removed entirely. The `IrodsHttpClient` constructor should accept a single URL instead.

Please open an issue for this. We can handle this in a separate PR.

Reply:
Created #44